Wikipedia/Localizations/ar.lproj/Localizable.strings

```diff
@@ -239,6 +240,7 @@
 "welcome-volunteer-send-usage-reports" = "إرسال تقارير الاستخدام";
 "welcome-volunteer-continue-button" = "استمرار";
 "welcome-volunteer-thanks" = "شكرا لك!$1";
```

Comment:
@niedzielski I checked this in the app and the substitution is happening correctly :)